### ISSUE TITLE
tmux commentstring

### ISSUE THREAD

**Issue by rainerborene** [sheerun/vim-polyglot#53]
consider adding this to `vim-polyglot`

```viml
au BufNewFile,BufRead .tmux.conf setlocal commentstring=#\ %s
```

**Comment by tsigo** [comment]
Alternatively we could switch the tmux plugin to [vim-tmux](https://github.com/tmux-plugins/vim-tmux) which supports commentstring correctly.

**Comment by sheerun** [comment]
vim-tmux does little to much. this is syntax-only plugin.

I've sent PRs to tmux plugins vim-polyglot uses: https://github.com/tejr/vim-tmux/pull/2

**Comment by sheerun** [comment]
I released v1.13.1 with this fix applied. Cheers
